### ISSUE TITLE
Remove page reload when creating one on one

### ISSUE THREAD

**Issue by PVince81** [nextcloud/spreed#4594]
## Steps to reproduce

1. Login as admin
1. Create another user "alice"
1. Create a group conversation and invite alice (and possibly others)
1. Open the participants tab
1. Click Alice's avatar and pick "Talk to Alice"

### Expected behaviour

No page reload, a new conversation opens.

### Actual behaviour

A full page reload happens, which feels a bit heavy for just creating a new conversation.

## Talk app

**Talk app version:** 11.0 pre-alpha git master fd02d1a41e89043582523d4945e68c9ab013bf46

**Custom Signaling server configured:** no

### Analysis

The reason why it happens is that the call URLs looks like "index.php/call/$token" but the URL that "Talk to Alice" points to is "index.php/apps/spreed/?callUser=alice".

One idea would be to try and unify those URLs, maybe it will become "index.php/call/?callUser=alice" with the goal of making it detectable as a route change. The router could then call the necessary services and then replace the URL with the one from the new room after creation (might need some waiting message while that URL is in place...).

This proposal would also move the logic to create a private conversation from the server to the browser.

**Comment by nickvergessen** [comment]
Pretty sure it's a duplicate, but couldn't find it.

The main issue is that the contacts/avatar menu does not allow apps to take over immediately, but triggers the page refresh itself.

**Comment by PVince81** [comment]
If they do trigger a page refresh, then it's an additional issue.

The link itself is pointing at a page not managed by the router, in a different path, so using that route would anyway trigger a page refresh.

**Comment by nickvergessen** [comment]
Yeah but talk can't say "okay i handle this request here and take care of the click on the action".
I think that was the missing part.

**Comment by ShGKme** [comment]
Seems to be fixed
